### DISCUSSION TITLE
Delete an old quarto pub site?

### DISCUSSION THREAD

**Discussion by JGradyH** [quarto-dev/quarto-cli#10849] [Q&A]
### Description

I deleted my quarto pub account, but my website persists on the internet.  I am currently replacing it with a new website with a custom domain, and I don't want confusion.  Is there a way to have my quarto pub site and blog completely removed without my account?  Sorry if this is the wrong place, but I have no idea where to put this question or how to solve this.

**Comment by cscheid** [comment]
I'll follow up with the quarto.pub folks at Posit and get back to you.

**Reply by JGradyH** [reply]
Thanks!

**Comment by kippandrew** [comment]
@JGradyH, do you mind sharing your account name so I can take a look at what the problem might be?

**Reply by cscheid** [reply]
(I can confirm that @kippandrew works at Posit, for everyone else's sake)

**Reply by JGradyH** [reply]
Hi @kippandrew - sorry but I don't remember exactly what the account name is... it might have been gheller? The site is at https://gheller.quarto.pub/grady-heller/index.html , and the blog is at the URL  https://gheller.quarto.pub/about.html.  Is that helpful?
Sorry for the mess!
